spec/factories.rb

```diff
@@ -275,6 +275,10 @@
     trait :part_time do
       study_mode { :part_time }
     end
+
+    trait :accredited_provider do
+      accredited_provider { provider }
+    end
```

Comment:
Optional: this could go into the `let` where we're using it, to avoid the factories from becoming too large.

Reply:
done 👍